trin-core/src/cli.rs

```diff
@@ -90,37 +90,44 @@ impl TrinConfig {
     {
         let config = Self::from_iter(args);
 
-        println!("Launching trin...");
-
         match config.web3_transport.as_str() {
             "http" => match &config.web3_ipc_path[..] {
-                DEFAULT_WEB3_IPC_PATH => {
-                    println!(
-                        "Protocol: {}\nWEB3 HTTP port: {}",
-                        config.web3_transport, config.web3_http_port
-                    )
-                }
+                DEFAULT_WEB3_IPC_PATH => {}
                 _ => panic!("Must not supply an ipc path when using http protocol for json-rpc"),
             },
             "ipc" => match &config.web3_http_port.to_string()[..] {
-                DEFAULT_WEB3_HTTP_PORT => {
-                    println!(
-                        "Protocol: {}\nIPC path: {}",
-                        config.web3_transport, config.web3_ipc_path
-                    )
-                }
+                DEFAULT_WEB3_HTTP_PORT => {}
                 _ => panic!("Must not supply an http port when using ipc protocol for json-rpc"),
             },
             val => panic!("Unsupported json-rpc protocol: {}", val),
         }
 
-        println!("Pool Size: {}", config.pool_size);
+        Ok(config)
+    }
+
+    pub fn display_config(&self) {
+        match self.web3_transport.as_str() {
+            "http" => {
+                println!(
+                    "Protocol: {}\nWEB3 HTTP port: {}",
+                    self.web3_transport, self.web3_http_port
+                )
```

Comment:
I'd also support if you wanted to upgrade these to info logs. We probably don't want any `println`s by the time we get to a stable release.